sql/walrus--0.1.sql

```diff
@@ -112,7 +56,20 @@ Validates that the user defined filters for a subscription:
 - values are coercable to the correct column type
 */
 declare
-    col_names text[] = cdc.selectable_columns(new.entity);
+    col_names text[] = coalesce(
+            array_agg(c.column_name order by c.ordinal_position),
+            '{}'::text[]
+        )
+        from information_schema.columns c
```

Comment:
Curious, does it have any effect when using `pg_attribute` instead?

Reply:
the pg_catalog tables do tend to respond faster than information_schema (haven't looked into why) but to get the same information there would need to be extra joins to filter the namespace by name vs oid

There's also a weird edge case with casting to regclass where, if the schema is in `search_path` it gets dropped from the regclass text representation 

e.g.
```sql
'public.my_table'::regclass => 'my_table`
```
which then makes it difficult to extract the schema reliably for filtering without an extra query